### ISSUE TITLE
Images: screen display

### ISSUE THREAD

**Issue by anadm** [SpeciesFileGroup/taxonworks#693]
See also https://github.com/SpeciesFileGroup/taxonworks/issues/691
When I open the image "show" screen, currently I see 3 links and the preview. 
This display could be improved as follows: 

- links on top (as they are now), followed by
![image](https://user-images.githubusercontent.com/1326839/49223840-13247d00-f3c7-11e8-94c3-68481f4d7268.png)
- Where both web previews (the jpg and the thumb) would appear side by side (the thumb not being a popup)
- Where the caption below each of the previews say what it is, followed by the address for use
- In this case the thumb on top of the page (which I assume is the web thumbnail) would become redundant (it's helpful in the search result table though, in the list screen)
- Rest of image info (md5 etc) in the bottom of the screen.
- Record housekeeping ok

**Comment by mjy** [comment]
@anadm This being the data (scaffolded) view we're limited to what we can customize.  I think what you are hinting at it perhaps the equivalent of a `Browse images` task?  We could customize that however needed, following pattern like Browse nomenclature, collection object, etc.

**Comment by anadm** [comment]
I am not sure what the path should be here, but definitely it would be more helpful to go to such an interface (e.g. when I click on the list) 
When you search OR browse the list of "images" --> get results --> go to image interface where:
- both web versions can be seen, 
- any of the addresses available to any versions can be copied
- large file can be downloaded
- rest of info regarding file can be seen (already available in bar)

I can see this happening for other kinds of files too (document, pdf etc.) so perhaps this can suggest a deeper change in data view? To me this does seem the right place.

**Comment by mjy** [comment]
OK, I'll see if we can make some tweaks to the attributes in Show.  Is the present image size for medium OK, or two big in your setup?

**Comment by anadm** [comment]
No, it seems about right. The thumbnail seems a bit small though. Perhaps 32x32 would appear better
